test/gtest/uct/test_peer_failure.cc

```diff
@@ -421,9 +421,9 @@ size_t test_uct_peer_failure_multiple::get_tx_queue_len() const
     size_t              tx_queue_len;
 
     if ((tl_name == "rc") || (tl_name == "rc_mlx5")) {
-        name = "RC_IB_TX_QUEUE_LEN";
-    } else if ((tl_name == "dc") || (tl_name == "dc_mlx5")) {
-        name = "DC_RC_IB_TX_QUEUE_LEN";
+        name = "RC_RC_IB_TX_QUEUE_LEN";
```

Comment:
why enough to use RC_TX_QUEUE_LEN?
maybe should split if to "rc" and "rc_mlx5"?

Reply:
> why enough to use RC_TX_QUEUE_LEN?
> why enough to use RC_TX_QUEUE_LEN?

do you mean "not enough ..."? if yes - since RC MLX5 and DC MLX5 use common RC MLX5 config (they inherits this config and it adds the extra "RC_" prefix), we should specify all prefixes, when requesting a config value.

> maybe should split if to "rc" and "rc_mlx5"?

good catch, it doesn't work for "rc", `RC_RC_*` is applicable only for RC MLX5.
fixed